src/app/components/list/overview/index.html

```diff
@@ -60,6 +60,120 @@
+      isOptional="true"
+    >
+    Specifies the name of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="label"
+      isOptional="true"
+    >
+    Specifies the text for the filter's summary item. If not specified, then the filter summary item text defaults to the <sky-code>value</sky-code> of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissable"
+      isOptional="true"
+    >
+    Indicates whether users can dismiss the filter's summary item. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="value"
+      isOptional="true"
+    >
+    Specifies the current value of the filter. This property accepts <sky-code>any</sky-code> value.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="defaultValue"
+      isOptional="true"
+    >
+    Specifies the default value of the filter. If the value of the filter is set to the default value, then the filter is not applied.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="filterFunction"
+      isOptional="true"
+    >
+    Specifies the function to apply to determine whether an item is filtered. This property is required when using in memory data provider. Acceptable values: <sky-code>(item: ListItemModel, filter: any) => boolean</sky-code>. This property accepts <sky-code>boolean</sky-code> values.    
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSearchModel properties">
+    <sky-demo-page-property
+      propertyName="searchText"
+      isOptional="true"
+    >
+    Specifies the text to search. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="functions"
+      isOptional="true"
+    >
+    Specifies an array of functions that should return a <sky-code>boolean</sky-code> value of <sky-code>true</sky-code> when the search is successful. Acceptable values: <sky-code>(data: any, searchText: string) => boolean></sky-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="fieldSelectors"
+      isOptional="true"
+    >
+    Specifies an array of <sky-code>string</sky-code> types which run against search fields. The <sky-code>string</sky-code> values should match the data's <sky-code>field</sky-code> value.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListSortModel properties">
+    <sky-demo-page-property
+      propertyName="available"
+      isOptional="true"
+    >
+    Specifies the list of avilable views to sort. Acceptables values: <sky-code>Array of ListSortLabelModel</sky-code>.
```

Comment:
"available" is misspelled. ... I'd change the last sentence to "This property accepts an array of `ListSortLabelModel` objects." And should we have another table with the properties of `ListSortLabelModel`? It seems like we should based on https://github.com/blackbaud/skyux-list-builder/blob/master/src/app/public/modules/list/state/sort/sort.model.ts. It looks to me like there are 5 properties there to document. (FWIW, I wouldn't necessarily worry about that for this PR. I'd just confirm with dev whether we should document those properties, and, if so, we can either handle it here or create a new docs issue to circle back to that.)

Reply:
Also, what does "views" mean here? What exactly is being sorted?

Reply:
This area gets super confusing for me too... I think this is an array of available sorts the end-user can apply. Not sure how to best put that in plain english. @Blackbaud-TrevorBurch am I reading this right?

Reply:
If that's the case, what are "available sorts"? I definitely think it would be good to explain whatever this is in plain English, but I don't really follow what we are doing here. Based on https://github.com/blackbaud/skyux-list-builder/blob/master/src/app/public/modules/list/state/sort/label.model.ts, I'm wondering if a "sort" is a pre-filtered version of the list, but I'm not sure.

Reply:
So this gets really tricky. That value really isn't used today. It appears to me that this is for `ListSortLabelModel` objects which are currently applied for the view being shown. This is for things such as when we have view switching and they can change the view from a grid to another view such as a repeater view (in the future). I'll be honest in that this only seems partially wired up when I dug into the code but in theory that is what this appears to be for. 

`global` is very similar but is those that are applied to all views. 

I'm honestly not sure the best way to put this as currently all sorts should show up as both global and available as I don't believe there is a current way to specify a non-global sort.

Reply:
If we aren't 100% confident with these properties, then should we just leave it out?

Reply:
I see the logic to leaving them out of the docs, but users could still find the properties and try to figure them out. If we have properties that aren't fully wired up or don't work, I think the responsible thing would be to be upfront about it. I think that for properties that are public and that we don't want users to use because they don't work, we should include a short boilerplate description that basically conveys that the properties are unfinished ("work in progress"?) and that we recommend against using them. What do you guys think?

Reply:
I'll do whatever y'all tell me to and what you think is best, ha. Personally, I'm leaning towards John's suggestion of documenting `ListSortLabelModel` and `ListFieldSelectorModel` with a caveat saying we don't recommend using them. I can be swayed by whatever will make the most sense for the consumer though.

Reply:
@Blackbaud-DeniseCisneros, can we resolve this? Or is anything still pending?